### PR TITLE
Login function

### PR DESCRIPTION
**Pull request by kmthorsnes**
#### Description
Changes proposed in this pull request:

* Adds ability for user to log in

#### What I have learned working on this feature: 
To create a PR as soon as a new branch and feature is started working on

#### Screenshots:
skjermbilde 2018-03-16 kl 11 41 13
skjermbilde 2018-03-16 kl 11 40 54

### CODE REVIEW

File: features/step_definitions/sign_in_steps.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  end
+end
+
+When("I fill in {string} with {string}") do |input, value|
+  fill_in input, with: value
+end
+
+Then("I should see {string}") do |string|
+
+end
```
Comment (faraznaeem) [inline features/step_definitions/sign_in_steps.rb]: empty step

File: features/user_can_sign_in.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  I should be able to log in
+
+  Background:
+    Given following user exist
+      | email             | password    |
+      | harald@norge.no   | OsloOslo123 |
+
+  Scenario: User successfully logs in
+    Given I visit the site
+    And I click "Sign in"
```
Comment (faraznaeem) [inline features/user_can_sign_in.feature]: These steps could be moved to the background

File: features/step_definitions/sign_in_steps.rb (hunk 1 above)
Comment (kmthorsnes) [reply features/step_definitions/sign_in_steps.rb]: Fixed

File: features/user_can_sign_in.feature (hunk 2 above)
Comment (kmthorsnes) [reply features/user_can_sign_in.feature]: Fixed

**Review by faraznaeem — APPROVED** [review]

**Comment by faraznaeem** [comment]
One of the team members also needs to approve this before merging

**Review by isabellelii — APPROVED** [review]
Looks good!

**Pull request merged**